ODIN_II/SRC/partial_map.cpp

```diff
@@ -221,11 +221,15 @@ void partial_map_node(nnode_t* node, short traverse_number, netlist_t* netlist)
             instantiate_multi_port_mux(node, traverse_number, netlist);
             break;
         case MULTIPLY: {
-            int mult_size = std::max<int>(node->input_port_sizes[0], node->input_port_sizes[1]);
-            if (hard_multipliers && mult_size >= min_mult) {
-                instantiate_hard_multiplier(node, traverse_number, netlist);
-            } else if (!hard_adders) {
-                instantiate_simple_soft_multiplier(node, traverse_number, netlist);
+            if (mixer->softenable(operation_list::MULTIPLY)) {
+                mixer->note_candidate_node(node, operation_list::MULTIPLY);
```

Comment:
don't use the enum name `operation_list::MULTIPLY` but use `MULTIPLY` to conform with the rest of the codebase